### PR TITLE
Updated works & PhD Thesis by Alzugaray

### PR DESCRIPTION
**Pull request by ialzugaray**
The PR contains the following changes:
- Moved BMVC'20 publication to generic feature detection and tracking section, as the tracking methodology is applicable to patch-features and not necessarily limited to corners (similar to 3DV'19).
- PhD thesis new entry (added as a link to relevant categories feature detection & tracking and corners).
- Updated links to code / video / presentation links for BMVC'20 / 3DV'19.

Please feel free to request any additional changes.

### CODE REVIEW

**Comment by likuui** [comment]
来信收到，谢谢！

File: README.md [hunk 1]
```diff
@@ -418,6 +421,8 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 - <a name="Mondal21iccvw"></a>Mondal, A., Shashant, R., Giraldo, J. H., Bouwmans, T., Chowdhury, A. S.,  
 *[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/papers/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
+- [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),
```
Comment (guillermogb) [inline README.md:424]: For some reason this link does not show up in the rich difference view of the pull request. I will check right after merging.

File: README.md [hunk 2]
```diff
@@ -467,6 +469,8 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 - <a name="Sengupta21ciss0"></a>Sengupta, J. P., Villemur, M., Andreou, A. G.,  
 *[Efficient, event-driven feature extraction and unsupervised object tracking for embedded applications](https://doi.org/10.1109/CISS50987.2021.9400234)*,  
 55th Annual Conf. on Information Sciences and Systems (CISS), 2021.
+- [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),
```
Comment (guillermogb) [inline README.md:472]: For some reason this link does not show up in the rich difference view of the pull request. I will check right after merging.

**Review by guillermogb — APPROVED** [review]
Thanks!

**Pull request merged**